diagnostic.md

```diff
@@ -45,6 +63,10 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
```

Comment:
We were just looking for CSS selectors here, not how to access the elements via jQuery, but I'm giving it to you anyway. ;)